app/Http/Controllers/Api/InventoryController2.php

```diff
@@ -0,0 +1,71 @@
+<?php namespace App\Http\Controllers\Api;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController2 extends Controller {
+
+	/**
+	 * Display the specified resource.
+	 *
+	 * @param  int  $tag
+	 * @return Json found $item
+	 */
+	public function show($tag)
+	{
+
+        $item=Item::where('asset_tag',$tag)->first();
+        return $item;
+
+	}
+
+
+	/**
+	 * Update the specified resource in storage.
+	 *
+	 * @param  int  $tag
+	 * @return Json updated $item
+	 */
+	public function update($tag)
+	{
+        $item=Item::where('asset_tag',$tag)->first();
+
+        if(is_null($item))
+        {
+            return  Response::json('Not Found',404);
+        }
+
+        $updateItem = Input::all();
+
+        //TODO:$updateItem should be validated
+
+        $item->fill($updateItem);
+        $item->save();
+        return $item;
+	}
+
+	/**
+	 * Remove the specified resource from storage.
+	 *
+	 * @param  int  $tag
+	 * @return Json removed $item
+	 */
+	public function destroy($tag)
+	{
+        $item=Item::where('asset_tag',$tag)->first();
+
+        if(is_null($item))
+        {
+            return  Response::json('Not Found',404);
+        }
+
+        $deleteItem=$item;
+        $item->delete();
+
+        return $deleteItem;
```

Comment:
It's arguable whether a DELETE request should return the thing it just deleted, but this is fine, I think.